assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -144,7 +144,11 @@ function DisplayElement({ element, previewMode, isAnimatable = false }) {
   });
 
   const bgOverlayRef = useRef(null);
-  useColorTransformHandler({ id, targetRef: bgOverlayRef });
+  useColorTransformHandler({
+    id,
+    targetRef: bgOverlayRef,
+    resetOnNullTransform: false,
```

Comment:
Why is this one needed? Not complaining, just want to understand the bug happening without it :)

Reply:
Yea, so the colorTransformHandler was weirdly being called with a null transform once at the very end of dragging & replacing a new image. This was that race condition I mentioned that caused the handler to clear the overlay node's styles once after react had rendered them (even though the overlay was present on the elements story data). 

I just added a flag for this specific instance to opt out of that behavior. I'm not in love with this solution, so all ears if you have anything better, but this worked and didn't seem to cause any harm